src/cmd/troubleshoot/image.go

```diff
@@ -210,7 +210,7 @@ func imageAvailable(httpClient *http.Client, imageUrl string, apiToken string) e
 }
 
 func registryAvailable(httpClient *http.Client, registry string, apiToken string) error {
-	statusCode, err := connectToDockerRegistry(httpClient, registryUrl(registry), apiToken)
+	statusCode, err := connectToDockerRegistry(httpClient, registryUrl(registry)+"/v2/", apiToken)
```

Comment:
would it make sense to have the `/v2/` included in `registryUrl`? the same string is also used in `manifestUrl`

Reply:
otherwise lgtm